quarkus.ls/com.redhat.quarkus.ls/src/main/java/com/redhat/quarkus/services/QuarkusValidator.java

```diff
@@ -40,12 +42,14 @@
 	private final List<Diagnostic> diagnostics;
 
 	private final QuarkusValidationSettings validationSettings;
+	private final Set<String> propertyNameSet;
 
 	public QuarkusValidator(QuarkusProjectInfo projectInfo, List<Diagnostic> diagnostics,
 			QuarkusValidationSettings validationSettings) {
 		this.projectInfo = projectInfo;
 		this.diagnostics = diagnostics;
 		this.validationSettings = validationSettings;
+		this.propertyNameSet = new HashSet<String>();

```

Comment:
rename propertyNameSet with existingProperties

Reply:
I have made the changes